test/templates/button-group/index.njk

```diff
@@ -1,129 +1,74 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
-{% from "../button/component.njk" import button %}
 {% from "./component.njk" import button_group %}
 
+
 {% block content %}
 <style media="screen">
   .fd-button-group {
     margin-bottom: 10px;
   }
 </style>
-<h2>default</h2>
 
+<style>
+  .btn-playground {
+    border-style: solid;
+    border-width: 1px;
+    border-radius: 4px;
+  }
+</style>
 
-    {% for size in ["","compact"] %}
-      {%- set modifier = { block: [size] } %}
-      {%- if size == "default" %}
-        {%- set modifier = { block: [] } %}
-      {%- endif %}
-<h2> {{size}}</h2>
+<h2>default</h2>
 
 {% set example %}
 
   {% call button_group() -%}
-    {{-  button(
-      { icon: 'survey' },
-        modifier={ block: [size] },
-        aria={ label: 'Survey' })
-      }}
-    {{-  button(
-        { icon: 'pie-chart' },
-          modifier={ block: [size] },
-          aria={ label: 'Chart', selected: true })
-        }}
-    {{-  button(
-          { icon: 'pool' },
-            modifier={ block: [size] },
-            aria={ label: 'Pool' })
-          }}
+    <button class="btn-playground sap-icon--survey" aria-label="Survey"></button>
+    <button class="btn-playground sap-icon--pie-chart" aria-label="Chart"></button>
+    <button class="btn-playground sap-icon--pool" aria-label="Pool"></button>
```

Comment:
These buttons should _not_ be using icon classes from the style library.  These "plain" buttons do not need to look like SAP buttons or look like previous playground examples.  These should literally be very simple-styled buttons that exist only to verify button group styles (_e.g._  border radius on or off, borders not doubling, etc.)